pkg/sql/catalog/descs/collection.go

```diff
@@ -1078,14 +1093,25 @@ func (tc *Collection) hydrateTypesInTableDesc(
 			}
 			name := tree.MakeNewQualifiedTypeName(dbDesc.Name, sc.Name, desc.Name)
 			return name, desc, nil
+		})
+
+		// Utilize the cache of hydrated tables if we have one.
```

Comment:
Add a comment about the fact that we might get rejected from the cache if we have an uncommitted descriptor, or if any of the types referenced are uncommitted too

Reply:
Done.